src/program/snabbvmx/lwaftr/setup.lua

```diff
@@ -0,0 +1,228 @@
+module(..., package.seeall)
+
+local PcapFilter = require("apps.packet_filter.pcap_filter").PcapFilter
+local S = require("syscall")
+local V4V6 = require("apps.lwaftr.V4V6").V4V6
+local VhostUser = require("apps.vhost.vhost_user").VhostUser
+local basic_apps = require("apps.basic.basic_apps")
+local bt = require("apps.lwaftr.binding_table")
+local config = require("core.config")
+local ipv4_apps = require("apps.lwaftr.ipv4_apps")
+local ipv6_apps = require("apps.lwaftr.ipv6_apps")
+local lib = require("core.lib")
+local lwaftr = require("apps.lwaftr.lwaftr")
+local lwcounter = require("apps.lwaftr.lwcounter")
+local nh_fwd = require("apps.nh_fwd.nh_fwd")
+local pci = require("lib.hardware.pci")
+local tap = require("apps.tap.tap")
+
+local yesno = lib.yesno
+
+-- TODO redundant function dir_exists also in lwaftr.lua
+local function dir_exists (path)
```

Comment:
Same reasoning as for `file_exists`. It make sense to fix this in another branch, but then please create an issue for it (if it's not already there) so that we don't forget.